src/applications/mhv/medications/containers/LandingPage.jsx

```diff
@@ -30,6 +109,38 @@ const LandingPage = () => {
     [dispatch],
   );
 
+  useEffect(
+    () => {
+      let arr;
+      if (prescriptions) {
+        arr = buildPrescriptionList();
+      }
+      setRxList(arr);
```

Comment:
```suggestion
      if (prescriptions) {
        setRxList(buildPrescriptionList());
      }
```

Would this make sense? Seems like the prescription list could get set to `undefined` is not truthy.

Reply:
i see yes that makes sense, I didn't think of that thanks I'll apply it!